### PR TITLE
Revert favicons to ownCloud color scheme

### PR DESCRIPTION
**Pull request by dlandon**

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <info>
 	<id>tasks</id>
 	<name>Tasks</name>
-	<version>0.9.5</version>
+	<version>0.9.5a</version>
```
Comment (DeepDiver1975) [inline appinfo/info.xml]: better use 0.9.6

Comment (dlandon) [reply appinfo/info.xml]: It won't track Nextcloud versions and there are no functional changes.

Comment (DeepDiver1975) [reply appinfo/info.xml]: 0.9.5a is not a valid version number - you cannot upload this to the marketplace

**Comment by dlandon** [comment]
Can I get a review of the code so I can release to the Marketplace?

**Pull request merged**